modules/client-configuration/pages/configuration-management.adoc

```diff
@@ -3,6 +3,12 @@
 
 You can use configuration files and channels to manage configuration for your clients, rather than configuring each client manually.
 
+[NOTE]
+====
+Some of the following features are available only for traditional clients.
```

Comment:
Hmm, this is for "master" and there we drop traditional stack.

Reply:
Yes, there are still many such references in master.  This file requires a general walkthru.  I'd like to backport (4.3) it first, and then prune this file (chapter).

Reply:
makes sense